gr-uhd/grc/uhd_rfnoc_null_src_sink.block.yml

```
@@ -0,0 +1,54 @@
+id: uhd_rfnoc_null_src_sink
+label: RFNoC Null Source Sink Block
+category: '[Core]/UHD/RFNoC/Blocks'
+
+parameters:
+- id: btype
+  label: Block Type
+  dtype: enum
+  options: ['sink', 'source']
```

Can you add a sink&source mode please?

```suggestion
  options: ['sink', 'source', 'both']
```

The reason being: If you wanted to use the same block for sink & source (which is totally fine from an RFNoC perspective) we currently have an issue where you can only access a block so many times from GNU Radio before gr-uhd becomes suspicious. That's something we need to improve elsewhere, but why not simply make this block accessible from both ends at once.

---

No problem at all! I have pushed the change.